### PR TITLE
Start trading period after deposit transaction confirmation

### PR DESCRIPTION
**Pull request by ManfredKarrer**
Hi Chris. Could you make a review?

### CODE REVIEW

File: gui/src/main/java/io/bisq/gui/main/offer/createoffer/CreateOfferViewModel.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ protected void activate() {
                 switch (BisqEnvironment.getBaseCurrencyNetwork().getCurrencyCode()) {
                     case "BTC":
                         amount.set("0.0001");
-                        price.set("14029");
+                        price.set("11029");
```
Comment (cbeams) [inline gui/src/main/java/io/bisq/gui/main/offer/createoffer/CreateOfferViewModel.java:201]: Why?

**Review by cbeams — APPROVED** [review]
Untested ACK.

Nit: The logic in the new `getTradeStartTime` would be simpler and clearer without the local `startTime` var, using `return` statements everywhere it was assigned.

Note: I've created bisq-network/docs#28 to capture style tips like the one above, and so am referencing it here.

**Comment by cbeams** [comment]
Sanity-check: the review comment I left above contained a reference to bisq-network/docs#28, just as this comment does, but no reference showed up in the timeline for that review comment. Does a reference show up now?

**Comment by cbeams** [comment]
It does show up now. That seems like a @GitHub bug.

File: gui/src/main/java/io/bisq/gui/main/offer/createoffer/CreateOfferViewModel.java (hunk 1 above)
Comment (ManfredKarrer) [reply gui/src/main/java/io/bisq/gui/main/offer/createoffer/CreateOfferViewModel.java:201]: I just adjusted to current btc price for dev testing with pre-filled values. The old value created a price> 10% distance thus require a manual entering of the price.

**Comment by ManfredKarrer** [comment]
The reasons why I used the local var was because it is used in the logging. 
I personally prefer only one return at the end of a method rather then several inside the methods branches. In such simple cases like here it might be irrelevant...

**Pull request merged**